test/test.toml

```diff
@@ -0,0 +1,21 @@
+# Visual test file.
+# You can run
+#
+#     nnoremap <F10> <cmd>echo synIDattr(synID(line('.'), col('.'), 1), 'name')<CR>
+#
+# and then press F10 to get the highlight group under the cursor.
+
+# issue 9
+issue9 = [
+    ["a", "b", "c"]
```

Comment:
Indent this by 2 spaces instead of 4, since that's what the indent in this plugin uses.

Reply:
Done, and I also added a modeline so anyone that edits gets 2 space indent automatically.